compiler/luci-interpreter/src/kernels/Elu.test.cpp

```diff
@@ -59,6 +59,35 @@ TEST(EluTest, SimpleElu)
       });
 }
 
+TEST(EluTest, InOutTypeMismatch_NEG)
+{
+  Shape input_shape{1, 2, 4, 1};
+  std::vector<float> input_data{
+      0, -6, 2,  -4,   //
+      3, -2, 10, -0.1, //
+  };
+  Tensor input_tensor = makeInputTensor<DataType::FLOAT32>(input_shape, input_data);
+  Tensor output_tensor = makeOutputTensor(DataType::U8);
+
+  Elu kernel(&input_tensor, &output_tensor);
+  EXPECT_ANY_THROW(kernel.configure());
+}
+
+TEST(EluTest, InvalidType_NEG)
```

Comment:
@s-barannikov 
a little bit question about this. Currently not support for this type, but later, this would be enabled for other types.
Should I leave this or remove this?

Reply:
I'd remove it. IMHO negative tests should cover _invalid_ cases, not _unsupported_.
The difference between _invalid_ and _unsupported_ can be vague, but this one is not the case :)

Reply:
Okay I will remove this! Thanks for comment!